src/test/java/org/lightcouch/tests/ViewsTest.java

```diff
@@ -95,6 +96,18 @@ public void byComplexKey() {
 	}
 
 	@Test
+	public void byComplexKeys() {
+		List<int[]> keysToGet = new Vector<int[]>();
```

Comment:
You are right. But as you said this is just a test so I'm not going to invest more time in it.